README.md

```diff
@@ -1,7 +1,28 @@
 # hewer
+v0.1.0
 
-### Need
+#### Installation Prerequisites
+1. Have go-lang installed ([duh](http://lmgtfy.com/?q=go-lang+install+download))
+1. Environment setup according to specs on [golang.org](https://golang.org/docs)
+  * [Workspace](https://golang.org/doc/code.html#Workspaces)
+  * [Environment variable(s)](https://golang.org/doc/code.html#GOPATH) - and don't forget to add `$GOPATH/bin` to your `PATH`
 
-### Some
+### Installation
 
-### Docs
+Install package via go tool
+```
+$ go get github.com/NexTraq/hewer
+```
+
+Verify Installation
+```
+$ hewer -v
+```
+
+### Usage
+
+```
+$ hewer <fileName> -k <keyName>
+```
+
+run `$ hewer help` for more information
```

Comment:
This should be `hewer --help`. Otherwise, it thinks you want to parse a file called "help".